files/build_templates/sonic_debian_extension.j2

```diff
@@ -90,6 +90,10 @@ sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/ifupdown2_*.deb || \
 sudo dpkg --root=$FILESYSTEM_ROOT -i $debs_path/iptables_*.deb || \
     sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
 
+# Install a more recent version of ntp  (and its dependencies via 'apt-get -y install -f')
+sudo dpkg --root=$FILESYSTEM_ROOT --force-confdef --force-confold -i $debs_path/ntp_*.deb || \
```

Comment:
I believe its the apt-get which is failing. We can either avoid it by installing missing dependencies(tried and working on the other PR) or specify --force-confold option to choose the old config file. I will try the --force-confold on the other PR

-    sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f
+    sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y \
+      -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" install -f